tests/integration/tools/impl/test_file_system.py

```
@@ -95,6 +95,48 @@ async def test_edit_file(create_test_graph, agent_context, temp_dir: Path):
+                        "name": "edit_file",
+                        "args": {
+                            "file_path": "edit_json.txt",
+                            "edits": '[{"old_content": "line 2", "new_content": "modified line 2"}]',
+                        },
+                    }
+                ],
+            ),
+        ],
+    }
+
+    result = await app.ainvoke(
```

Ruff (0.14.5)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)